docs/README.md

```diff
@@ -33,42 +37,736 @@ All output will adhere to the same general JSON structure:
+| meta          |           | CSV of metadata fields to include [Learn More](#metadata)  |
+| status        | Published | CSV of statuses [Learn More](#status)  |
+| lang          | *         | Include translation(s) [Learn More](#language)  |
+
+#### Possible Results
+| Code             | Description                                        |
+|------------------|----------------------------------------------------|
+| 200 OK           | `data`: Retrieved item<br>`meta`: Depends on requested metadata  |
+| 404 Not Found    | `message`: Collection doesn’t exist, or item doesn't exist within collection    |
+
+#### Examples
+
+* Return the project item with an ID of `1`
+  ```bash
+  curl -g https://api.directus.io/_/items/projects/1
+  ```
+
+
+
+
+### Get Items
+Gets items from within a given collection
+
+```http
+GET /items/[collection-name]/
+```
+
+#### Query Parameters
+| Name          | Default   | Description                                      |
+|---------------|-----------|--------------------------------------------------|
+| limit         | 200       | The number of items to request                   |
+| offset        | 0         | How many items to skip before fetching results   |
+| sort          | id        | CSV of fields to sort by [Learn More](#sorting)  |
+| fields        | *         | CSV of fields to include in response [Learn More](#fields)  |
+| filter[field] |           | Filter items using operators [Learn More](#filtering)  |
+| meta          |           | CSV of metadata fields to include [Learn More](#metadata)  |
+| status        | Published | CSV of statuses [Learn More](#status)  |
+| lang          | *         | Include translation(s) [Learn More](#language)  |
+| q             |           | Search string [Learn More](#search-query)  |
+| id            |           | CSV of primary keys to fetch  |
+
+#### Possible Results
+| Code             | Description                                        |
+|------------------|----------------------------------------------------|
+| 200 OK           | `data`: Array of items<br>`meta`: Depends on requested metadata  |
+| 404 Not Found    | `message`: Collection doesn’t exist    |
 
 #### Examples
 
 * Search for all projects in the `design` category
   ```bash
-  curl -g https://demo-api.getdirectus.com/_/items/projects?filter[category][eq]=design
+  curl -g https://api.directus.io/_/items/projects?filter[category][eq]=design
+  ```
+
+
+
+
+### Update Item
+Update a single item from within a given collection
+
+```http
+PATCH /items/[collection-name]/[primary-key-value]
+PUT /items/[collection-name]/[primary-key-value]
+```
+
+::: warning PATCH VS PUT
+* **PATCH** partially updates the item with the provided data, any missing data is ignored
+* **PUT** fully updates the item and any missing data will fallback to its default value
+:::
+
+
+#### Body
+A single item to be updated. Field keys must match the collection's column names
+
+#### Possible Results
```

Comment:
I'm not sure if the term `Possible Results` is the best.. For all we know, the API might return a 500 or other due to unexpected reasons. These results are the _most likely_ results..

Reply:
Also, 403 is a possible result for _all_ endpoints, seeing that the "public"-ness of an endpoint is set by the permissions, any endpoint can potentially return a 403 both on initial request as on the return value. 

F.e. if your user has the right to update, the update itself will go through, but it will still return 403..

Reply:
Which makes me think, in case you **don't have read access** but **do have update** access, we should probably return a response in the 2xx-range